### DISCUSSION TITLE
How can we help?

### DISCUSSION THREAD

**Discussion by plve** [jonas-hoebenreich/wc-donation-platform#74] [General]
I deeply believe in this project. From finding more people to help to fundraising, I want to be part and help this project became the best it can be!

**Comment by jonas-hoebenreich** [comment]
Hi @plve,

thanks for reaching out.
A good start would be checking my [roadmap](https://www.wc-donation.com/documentation/support/roadmap/) and getting familiar with the plugin. You are always welcome to open Pull Requests and contribute to the project :)

Best,
Jonas
